extension/js/common/message-renderer.ts

```diff
@@ -521,7 +521,8 @@ export class MessageRenderer {
     sigResult: VerifyRes | undefined,
     renderModule: RenderInterface,
     retryVerification: (() => Promise<VerifyRes | undefined>) | undefined,
-    plainSubject: string | undefined
+    plainSubject: string | undefined,
+    isCheckSumInvalid = false
```

Comment:
let's call it `isChecksumInvalid` instead of `isCheckSumInvalid`, as `checksum` is a single word, and we already call it this way in `validateChecksum`

Reply:
Sounds good!